demo/config_actin.yml

```diff
@@ -18,3 +18,11 @@ taxon_focus: 'euk'
 uniprot_additional_fields: []
 min_length: 0
 max_length: 0
+
+# Resource execution settings
+resources:
```

Comment:
These changes should probably also be added to `config.yml`, since this config file is meant just for the actin demo.

Reply:
I think @lebovic is only using the actin demo file, so this should be functional for him. We can either proceed with this as is and post an issue to add resources to config.yml, or we can add it in to this PR. Do you have a pref @mezarque?

Reply:
If we expect these kinds of parameters to be useful (and not interfere with the standard use case, which is just to run things on a personal computer), then it might be good to include in the main config file. I imagine that `cores` and `local-cores` could cause issues depending on a user's machine though? Does `snakemake` fail if there are not enough cores, or does it just scale down?

Reply:
let's leave it out for now and create an issue. it should not interfere with standard use case on a personal computer, you usually have to tell snakemake to use this info (at least when it's specified as a directive in a rule), but i'd  like to double check that before adding. I'll make an issue.

Reply:
Sorry, I should have included more context in this PR! The intent was just to create a working test set for parallelization with the `demo/config_actin.yml` config – intentionally not modifying the base `config.yml`.

That said, it should be okay to include in `config.yml` – but maybe with different settings. Directive-by-directive, here's what I think:

### `resources [...]`

Might interfere with the standard use-case?

By default, Snakemake allocates 1 vCPU and 1 GB of RAM per task. This was included to increase the default allocation for remote tasks; it might be suboptimal if the local machine has less. Iirc, this uses the minimum of actual vs. requested – but worth testing.

### `max-jobs-per-second: 1`

Won't interfere with the standard use-case.

This is the maximum number of jobs *spawned* per second. This doesn't apply to local runs, only remote. Default is 10, a setting of 1 oddly speeds up execution for remote execution if there's a massive burst of seconds-long tasks (e.g. `download_pdbs` in this pipeline).

### `max-status-checks-per-second: 10`

Won't interfere with the standard use-case.

This is actually the default value, but I left it in for clarity. Feel free to remove! If there's a high value here (e.g. 1000), it slows down the pipeline execution (also limited by `download_pdbs in this pipeline).

### `local-cores: 10`

Shouldn't interfere with the standard use-case.

It's only used for cloud/cluster mode, not when running locally. It's meant to limit CPU usage on a "head node" for a cluster, or similar paradigm for cloud execution.